Data/Multibase.idr

```diff
@@ -0,0 +1,59 @@
+
+module Data.Multibase
+
+import Data.Fin
+import Data.Vect
+
+%default total
+
+%access export
+
+public export
+data BaseSymbol : Nat -> Type where
+  SBase1     : BaseSymbol 0
+  SBase2     : BaseSymbol 1
+  SBase8     : BaseSymbol 7
+  SBase10    : BaseSymbol 9
+  SBase16    : BaseSymbol 15
+  SBase32    : BaseSymbol 31
+  SBase58btc : BaseSymbol 57
+  SBase64    : BaseSymbol 63
+  
+public export
+record MultibaseDigest (length : Nat) (n : Nat) where
```

Comment:
This seems to be the core type of the whole thing. Would it be helpful to add a doc comment with an example? And perhaps a link to something like https://github.com/multiformats/multihash#format?

Reply:
Good idea! Will do